aspnetcore/tutorials/first-mvc-app/adding-controller.md

```diff
@@ -145,7 +145,7 @@ In the preceding URL:
 
 In the preceding example:
 
-* The third URL segment matched the route parameter `id`.
+* The third URL segment matched the route parameter `id` as defined in the routing format in the `Program.cs` file.
```

Comment:
```suggestion
* The third URL segment matched the route parameter `id` as defined in the routing template in the `Program.cs` file.
```

Reply:
Typically it is referred to either as the routing template or routing pattern.  In this case, routing template is best.